sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -81,34 +92,39 @@ export interface DeletedKey extends Key {
     readonly scheduledPurgeDate?: Date;
 }
 
+// Warning: (ae-forgotten-export) The symbol "DeleteKeyPollOperationState" needs to be exported by the entry point index.d.ts
```

Comment:
missing export?

Reply:
I actually don't want this to be exported. Should we export it?